Login/src/main/java/com/bms/login/controllers/TestController.java

```diff
@@ -0,0 +1,32 @@
+package com.bms.login.controllers;
+
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/api/test")
+public class TestController {
```

Comment:
Change the name of the class to something like MockController as TestController is confusing and not being used for testing purpose

Reply:
I don't even know if its needed. Changing it anyway